Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM node:10
```

Comment:
In this Dockerfile, I'm installing supervisord which is a python package to manage running the vue app behind nginx. If you know of a package that's native to node that we can use instead, I'm happy to try that out because this definitely bloats the image a bit.